.travis.yml

```diff
@@ -4,8 +4,6 @@ language: java
 jobs:
   fast_finish: true
   include:
-  - os: linux
-    jdk: openjdk8
   - os: linux
```

Comment:
My understanding is that we should not remove JDK 8 for building. We removed it just as runtime.